extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -158,8 +158,12 @@ export class DoubleclickA4aEligibility {
     }
     if (useRemoteHtml) {
       warnDeprecation('remote.html');
+      if (!element.getAttribute('rtc-config')) {
```

Comment:
I'm sure we talked about this and I realize its unrelated to this PR but why are we warning when rtc config is present?  Seems that this is sufficient:

if (useRemoteHtml && !element.getAttribute('rtc-config')) {
     warnDeprecation('remote.html');
     element.setAttribute('data-ati', '1');
     return false;
}

Reply:
Basically what we had discussed offline about this was that it doesn't hurt to make the visibility about this more widespread, and also this should help them realize that if both are on their page, they should really just be outright deleting remote